### PR TITLE
Clean up contributors.md

### PR DESCRIPTION
**Pull request by deissnerk**
As decided in the weekly call, I removed the company names and put the names in alphabetical order.

### CODE REVIEW

**Review by markpeek — APPROVED** [review]
Looks good except for some extra blank lines at the end. Thanks for taking care of this task.

**Comment by duglin** [comment]
LGTM thanks!

**Review by jskeet — APPROVED** [review]
LGTM modulo the blank lines.

Is Jeremy Coffield (currently no link) https://github.com/plfx? Seems likely. I haven't looked into the other link-less names.

**Comment by deissnerk** [comment]
I removed the empty lines at the end. Yes, there were a few names without links. I did not try to add links to avoid confusion. If somebody provides the correct github links, I will add them.

**Comment by deissnerk** [comment]
Thanks for bringing this whole issue to my attention @devguyio ! 😄

**Comment by jskeet** [comment]
> Yes, there were a few names without links. I did not try to add links to avoid confusion.

I think that's reasonable. I guess if folks want to be linked, they can create their own PRs :)

File: docs/contributors.md [hunk 1]
```diff
@@ -13,69 +13,95 @@ to see who has been involved already.
...
+- Daniel Krook - [@krook](https://github.com/krook)
+
+# L
+- Rob Leidle
+- David Lyle - [@dklyle](https://github.com/dklyle)
+
+# M
+- Mike McDonald
+- Gunnar Morling - [@gunnarmorling](https://github.com/gunnarmorling/)
+- Rachel Myers - [@rachelmyers](https://github.com/rachelmyers)
```
Comment (Cali0707) [inline docs/contributors.md:65]: Not sure if this PR is the right place to add myself (I'm happy to open a separate PR if that's better)
```suggestion
- Rachel Myers - [@rachelmyers](https://github.com/rachelmyers)
- Calum Murray - [@Cali0707](https://github.com/Cali0707/)
```

**Comment by duglin** [comment]
Approved on the 7/18 call

**Pull request merged**